### ISSUE TITLE
quiet_assets was resolved to 1.0.0 getting this isue while upgrading to rails 5.0.0

### ISSUE THREAD

**Issue by jawadakram20** [evrone/quiet_assets#53]

**Comment by route** [comment]
wat?

**Comment by Deekor** [comment]
Same here: 

```
Bundler could not find compatible versions for gem "rails":
  In Gemfile:
    rails (= 5.0.1)

    quiet_assets was resolved to 1.0.0, which depends on
      rails (~> 3.1)

**Comment by route** [comment]
@Deekor just throw `quiet_assets` out of a Gemfile and see Deprecation line in the README
